submodules/LIB/hardware/syn/build.tcl

```diff
@@ -24,8 +24,10 @@ set_db init_hdl_search_path $INCLUDE
 set_db stdout_log genus.log
 set_db information_level 7
 set_db super_thread_debug_directory st_part_log
-
-set_db hdl_error_on_latch true 
+set_db hdl_error_on_latch true
```

Comment:
is this needed? Isn't true the default?

Reply:
The original code already had it set as true. I didn't even think about that.

Reply:
Ok, but I still don't know the answer

Reply:
i just tested, the standart the latch error is false. i removed the set from the syn_build.tcl and removed from the set from the build tcl, and it gave no synthesis warnings.